1-exercises/B-object-arrays/exercise-1.js

```diff
@@ -6,14 +6,14 @@
    hint: use the array "map" method.
 
 */
-
+ 
 let people = [
   { name: "Bob", age: 30 },
   { name: "John", age: 20 },
   { name: "Elise", age: 35 },
 ];
-
-let peopleNames =
+ 
+let peopleNames = people.map( x => { return x.name } )
```

Comment:
Very good. Think about variable names. Perhaps this would read better. You can also shorten the code as yo are using 'arrow syntax'

let peopleNames = people.map((person) => person.name);